apps/condo/domains/resident/schema/DiscoverServiceConsumersService.js

```diff
@@ -477,7 +514,16 @@ const DiscoverServiceConsumersService = new GQLCustomSchema('DiscoverServiceCons
                         })
 
                         const data = {
-                            ...serviceConsumerData,
+                            ...pick(serviceConsumerData, [
+                                'dv',
+                                'sender',
+                                'resident',
+                                'accountNumber',
+                                'organization',
+                                'billingAccount',
+                                'billingIntegrationContext',
+                                'acquiringIntegrationContext',
```

Comment:
Why do you need to explicitly collect fields from serviceConsumerData?

Reply:
Yes, because `data` now contains some additional fields needed for this mutation, but erroneous for creating a service consumer.